Pod/Classes/Categories/NotificationCenter+PostContentSizeCategoryChange.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  NotificationCenter+CustomPosts.swift
```

Comment:
Do we want to leave those headers in files?

Reply:
Some CIs requires credentials with specific number of lines at the beginning of the file.
Beside I don't know if we should remove or not those headers as a part of Allegro common Open Source police. 
But all files requires `Grupa Allegro` -> `Allegro` change, so we can check our doubts later and change everywhere.